### ISSUE TITLE
Adapt control structures to use short statements where applicable

### ISSUE THREAD

**Issue by connorkuehl** [connorkuehl/popple#20]
In Go, a statement can precede a conditional in an if-statement. Refactor structures like this to use those to constrain the scope of variables to the if-else-blocks where appropriate.

I.e.,

```diff
diff --git a/command.go b/command.go
index 3dc4f75..94da1a5 100644
--- a/command.go
+++ b/command.go
@@ -46,8 +46,7 @@ func CheckKarma(ctx *Context) {
                sep = " "
        }
 
-       _, err := s.ChannelMessageSend(m.ChannelID, reply.String())
-       if err != nil {
+       if _, err := s.ChannelMessageSend(m.ChannelID, reply.String()); err != nil {
                log.Printf("Error when sending reply to channel: %s\n", err)
        }
 }
```

**Comment by krishnaindani** [comment]
hey, could I grab this to work on I would like to try?

**Comment by connorkuehl** [comment]
Sure! Thanks. Let me know if I can help out at all.

**Comment by krishnaindani** [comment]
sure thanks, I don't see a whole lot of changes. I will push the PR.

**Comment by connorkuehl** [comment]
Closed by e1dd988
